source/hardware/controllers/ion_discover_dj_pro.rst

```diff
@@ -0,0 +1,99 @@
+ION Discover DJ Pro
+===================
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro
+
```

Comment:
We don't have permission to use this image and it's also low-res and doesn't add information. Please remove (and also the image file).

Reply:
Done